lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -206,11 +249,11 @@ def make_vetext_request(request_body):
             timeout=HTTPTIMEOUT,
             headers=headers
         )
-        logger.info('VeText POST complete')
+        logger.info("VeText POST complete")
         response.raise_for_status()
         
         logger.info("VeText call complete with response: %d", response.status_code)
-        logger.debug("VeText response: %s", response.content)
+        logger.info("VeText response: %s", response.content)
```

Comment:
again, probably best left at debug

Reply:
again, please do not resolve convos without an explanation. think this should still be debug,